CHANGES

```diff
@@ -1,3 +1,7 @@
+0.10.11
+
+- Better support for Django 1.6 and Django 1.7
```

Comment:
We should bump this to 0.11 or something, and say that we've dropped support for older Django too. I think that's worth a larger version bump.